spec/lib/kafka/avro_producer_spec.rb

```diff
@@ -44,25 +46,57 @@
   end
 
   context 'producing a message successfully' do
+    let(:topic1_payload_value) { "\x00\x00\x00\x00\x05\x02\x06key\nvalue\x00" }
```

Comment:
make sure the hardcoded and dynamically fetched schema payload get parsed the same